### PR TITLE
[Bug]fix the calculation of the "_start_trash_sweep" run interval.

### LINKED ISSUE #6166: [BUG] The calculation of the "sweep garbage" interval is wrong
The maximum garbage cleaning interval in the current code  is not work. Because the interval for garbage cleaning is related to disk usage, and this variable will become larger and larger as it runs
addfb6bf8a9d2321e69676aa46968c14
6c409aab9dc9e291b67e7bba05e38146

### PR DESCRIPTION
**Pull request by BiteTheDDDDt**
## Proposed changes

Describe the big picture of your changes here to communicate to the maintainers why we should accept this pull request. If it fixes a bug or resolves a feature request, be sure to link to that issue.

## Types of changes

What types of changes does your code introduce to Doris?
_Put an `x` in the boxes that apply_

- [X] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation Update (if none of the other choices apply)
- [ ] Code refactor (Modify the code structure, format the code, etc...)

## Checklist

_Put an `x` in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your code._

- [X] I have created an issue on (Fix #6166) and described the bug/feature there in detail
- [ ] Compiling and unit tests pass locally with my changes
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] If these changes need document changes, I have updated the document
- [ ] Any dependent changes have been merged

## Further comments

If this is a relatively large or complex change, kick off the discussion at dev@doris.apache.org by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

File: be/src/olap/storage_engine.cpp [hunk 1]
```diff
@@ -681,6 +686,10 @@ OLAPStatus StorageEngine::_start_trash_sweep(double* usage) {
     return res;
 }
 
+OLAPStatus StorageEngine::start_trash_sweep() {
```
Comment (yangzhg) [inline be/src/olap/storage_engine.cpp:689]: why add this function

**Review by yangzhg — COMMENTED** [review]
tranlate  those comment into English

File: be/src/olap/storage_engine.cpp (hunk 1 above)
Comment (BiteTheDDDDt) [reply be/src/olap/storage_engine.cpp:689]: add a public function for users actively clean.

**Comment by BiteTheDDDDt** [comment]
> 
> 
> tranlate those comment into English

done.

**Comment by yangzhg** [comment]
you should create an ISSUE to describe this bug

File: be/src/olap/storage_engine.cpp (hunk 1 above)
Comment (yangzhg) [reply be/src/olap/storage_engine.cpp:689]: since that function is not used by this pr, please remove this

**Review by yangzhg — APPROVED** [review]
LGTM

**Pull request closed without merging**